sample/website_sample.py

```diff
@@ -44,13 +44,13 @@
     raise ValueError('Please provide parameter file with account information.')
 
 PORT = 8088
-TEMPLATE_AUTHZ_URL = ('https://login.windows.net/{}/oauth2/authorize?'+ 
+TEMPLATE_AUTHZ_URL = ('https://login.windows.net/{}/oauth2/authorize?'+
                       'response_type=code&client_id={}&redirect_uri={}&'+
                       'state={}&resource={}')
 RESOURCE = '00000002-0000-0000-c000-000000000000' #Graph Resource
```

Comment:
to be consistent, you should make the same change by reading the resource from the parameter file